app/src/main/res/values-v21/styles.xml

```diff
@@ -4,6 +4,6 @@
         <item name="windowActionBar">false</item>
         <item name="windowNoTitle">true</item>
         <item name="android:windowDrawsSystemBarBackgrounds">true</item>
-        <item name="android:statusBarColor">@android:color/transparent</item>
+        <item name="android:statusBarColor">@color/colorPrimaryDark</item>
```

Comment:
Is this change necessary?

Reply:
it was actually showing me transparent view on my Lollipop device.